frontend/src/components/nodes/NodesMap.tsx

```diff
@@ -391,6 +345,7 @@ export default class extends React.Component<State> {
         }}
       />
     );
+    let value = this.context;
```

Comment:
This does not make the code any better. Let's either use `const { nodeInfo } = this.context` or just use `this.context.nodeInfo` in place without renaming it to "value"